fvm/src/state_tree.rs

```diff
@@ -31,8 +32,10 @@ pub struct StateTree<S> {
     version: StateTreeVersion,
     info: Option<Cid>,
 
-    /// State cache
+    /// State snapshots.
     snaps: StateSnapshots,
+    /// Read-cache (always check the snapshots first)
+    read_cache: RefCell<HashMap<ActorID, Option<ActorState>>>,
```

Comment:
Improving the docs. Our future selves will appreciate it.

```suggestion
    /// Read-cache for the backing blockstore. This DOES NOT contain
    /// mutations in progress (contained in the snapshots). Always check
    /// snapshots first, then the read cache, then fall back to the backing
    /// blockstore.
    read_cache: RefCell<HashMap<ActorID, Option<ActorState>>>,
```